packages/frontend/src/hooks/pivotTable/pivotQueryResults.ts

```diff
@@ -3,21 +3,23 @@ import {
     MetricQuery,
     PivotConfig,
     PivotData,
+    PIVOT_TABLE_MAX_COLUMN_LIMIT,
     ResultRow,
     ResultValue,
 } from '@lightdash/common';
 import { isNumber } from 'lodash-es';
 import { Entries } from 'type-fest';
 
-const MAX_COLUMNS = 60;
-
 type PivotQueryResultsArgs = {
     pivotConfig: PivotConfig;
     metricQuery: Pick<
         MetricQuery,
         'dimensions' | 'metrics' | 'tableCalculations' | 'additionalMetrics'
     >;
     rows: ResultRow[];
+    options?: {
+        maxColumns: number;
+    };
```

Comment:
imo, having this as optional and defaulting to a constant is going to cause bugs in the future. 
eg: tomorrow someone implements CSV download with pivot data, uses this function but doesn't set the limit. 

I would change the code to require a limit and delete the constant from the codebase so the only place that defines the limit is in the lightdash config.

Reply:
yeah, makes sense. I'll update